yellowbrick/text/postag.py

```diff
@@ -114,251 +115,381 @@ def __init__(
+                            "other": 0,
+                        }
+        if self.stack:
+            penn_tag_map={}
+            for i in self.labels_:
+                penn_tag_map[i]=default_dic.copy()
+            return penn_tag_map
+        else:
+            return default_dic
 
     def _uni_tag_map(self):
         """
         Returns a Universal Dependencies part-of-speech tag map.
         """
-        return {
-            "noun": 0,
-            "verb": 0,
-            "adjective": 0,
-            "adverb": 0,
-            "adposition": 0,
-            "determiner": 0,
-            "pronoun": 0,
-            "conjunction": 0,
-            "infinitive": 0,
-            "punctuation": 0,
-            "number": 0,
-            "interjection": 0,
-            "symbol": 0,
-            "other": 0,
-        }
-
-    def _handle_universal(self, X):
-        """
-        Scan through the corpus to compute counts of each Universal
-        Dependencies part-of-speech.
-
-        Parameters
-        ----------
-        X : list or generator
-            Should be provided as a list of documents or a generator
-            that yields a list of documents that contain a list of
-            sentences that contain (token, tag) tuples.
-        """
-        jump = {
-            # combine proper and regular nouns
-            "NOUN": "noun", "PROPN": "noun",
-            "ADJ": "adjective",
-            "VERB": "verb",
-            # include particles with adverbs
-            "ADV": "adverb", "PART": "adverb",
-            "ADP": "adposition",
-            "PRON": "pronoun",
-            "CCONJ": "conjunction",
-            "PUNCT": "punctuation",
-            "DET": "determiner",
-            "NUM": "number",
-            "INTJ": "interjection",
-            "SYM": "symbol",
-        }
-
-        for tagged_doc in X:
-            for tagged_sent in tagged_doc:
-                for _, tag in tagged_sent:
-                    if tag == "SPACE":
-                        continue
-                    self.pos_tag_counts_[jump.get(tag, "other")] += 1
-
-    def _handle_treebank(self, X):
+        default_dic = {
+                        "noun": 0,
+                        "verb": 0,
+                        "adjective": 0,
+                        "adverb": 0,
+                        "adposition": 0,
+                        "determiner": 0,
+                        "pronoun": 0,
+                        "conjunction": 0,
+                        "infinitive": 0,
+                        "punctuation": 0,
+                        "number": 0,
+                        "interjection": 0,
+                        "symbol": 0,
+                        "other": 0,
+                    }
+        if self.stack:
+            uni_tag_map={}
+            for i in self.labels_:
+                uni_tag_map[i]=default_dic.copy()
+            return uni_tag_map
+        else:
+            return default_dic
+
+    def _handle_universal(self, X,y):
```

Comment:
The signature here should be `def _handle_universal(self, X, y=None)` to match the signature of `fit()`. Also would you mind adding the `y` parameter to the docstring under X?

Reply:
Bump!